CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Clarify documentation for --scan-host parameter [#1277](https://github.com/greenbone/gvmd/pull/1277)
 - In result iterator access severity directly if possible [#1321](https://github.com/greenbone/gvmd/pull/1321)
 - Change SCAP and CERT data to use new severity scoring [#1333](https://github.com/greenbone/gvmd/pull/1333) [#1357](https://github.com/greenbone/gvmd/pull/1357) [#1365](https://github.com/greenbone/gvmd/pull/1365)
+- The internal list of current Local Security Checks for the 'Auto-FP' and 'Closed CVEs' features was updated [#1381](https://github.com/greenbone/gvmd/pull/1381)
```

Comment:
```suggestion
- The internal list of current Local Security Checks for the 'Closed CVEs' features was updated [#1381](https://github.com/greenbone/gvmd/pull/1381)
```

Since Auto-FP is gone in 21.04 (cf. #1300) this don't need to be mentioned.